.github/workflows/run_periodic_tests.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: |
           sudo apt-get update
-          sudo apt install gfortran gcc libopenblas-dev graphviz
+          sudo apt install gfortran gcc libopenblas-dev graphviz pandoc
```

Comment:
Will this be required while building the docs locally, or is it only required for deployment? We might want to add this to the dev docs if it is an absolute requirement.

Reply:
Ah, great!